units/user/xdg-desktop.target

```diff
@@ -0,0 +1,14 @@
+#  SPDX-License-Identifier: LGPL-2.1+
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+[Unit]
+Description=Request startup of XDG autostart applications
```

Comment:
"Startup of XDG autostart applications".
This will appear as "Reached target ...", and by then "request" doesn't apply anymore.